TicTacToeForBot.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[52]:
```

Comment:
Remove unnecessary comments, please, also mind the needed number of blank lines.

Reply:
Done